components/kyma-environment-broker/internal/storage/driver/postsql/operation.go

```diff
@@ -551,5 +555,25 @@ func operationToDB(op *internal.Operation) dbmodel.OperationDTO {
 		CreatedAt:         op.CreatedAt,
 		Version:           op.Version,
 		InstanceID:        op.InstanceID,
+		OrchestrationID:   orchestrationIDtoSQL(op.OrchestrationID),
+	}
+}
+
+func orchestrationIDtoSQL(orchestrationID string) sql.NullString {
+	result := sql.NullString{}
+
+	if orchestrationID != "" {
+		result.String = orchestrationID
+		result.Valid = true
 	}
+
+	return result
+}
+
+func orchestrationIDtoString(orchestrationID sql.NullString) string {
+	if orchestrationID.Valid {
+		return orchestrationID.String
+	}
+
+	return ""
```

Comment:
What do you think about moving those functions to a common plane and calling them something like `StringToSQLNullString` and `SQLNullStringToString`? I think this logic is not exclusive to orchestrationID and can be reused